server/routes/games.js

```diff
@@ -49,7 +49,7 @@ router.get('/search', async (req, res, next) => {
  * GET /api/games/
  * */
 router.get('/', async (req, res, next) => {
-    res.json(await Game.find().populate('categories likes'))
+    res.json(await Game.find().populate(' likes'))

```

Comment:
Is the space in front of the likes intentional?

Reply:
Yes and no.  Just thought it might help me remember to put it back later.

Really was a mistake.